fintech-examples/fintech-ui/src/app/bank/list-accounts/list-accounts.component.ts

```diff
@@ -33,7 +33,9 @@ export class ListAccountsComponent implements OnInit {
 
   private loadAccount(): void {
     const settings = this.storageService.getSettings();
-    this.aisService.getAccounts(this.bankId, settings.loa, settings.withBalance).subscribe((response) => {
+    const online = !this.storageService.isAfterRedirect() && !settings.cacheLoa;
```

Comment:
isAfterRedirect does not exist any more and must not be used any more

Reply:
I added it in order to detect second call to list accounts or list transactions. 
If 'Use protocol cache' not checked then first call to list accounts/transactions will be with parameter online=true (it will update cache), second call will be online=false (it will return actually accounts or transactions).
All this relevant only for HBCI. In xs2a online param ignored.